### ISSUE TITLE
Visualization not working

### ISSUE THREAD

**Issue by cdunford** [kolofordjango/kolo#43]
When I click on frames or visualization, the visualization pane opens up but it's completely blank:

![image](https://user-images.githubusercontent.com/1405868/196826859-e6216c09-40c4-4070-8fc8-276b7b68c4c9.png)

Everything else seems to be working correctly; any idea what's going on?

**Comment by wilhelmklopp** [comment]
@cdunford thanks for opening this issue, very strange indeed!

With the visualisation in focus, could you open the webview developer tools via CMD/CTRL-SHIFT-P:

<img width="590" alt="image" src="https://user-images.githubusercontent.com/7718702/197015645-084765f5-7de7-496c-82a6-ad2be1f8a515.png">


and let me know if there are any error messages in the console?

**Comment by cdunford** [comment]
Seeing this:

![image](https://user-images.githubusercontent.com/1405868/197200618-2cd59d5e-fea5-44c9-92e0-78e3d9551961.png)


```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'start')
    at main.js:2:147330
    at ko (main.js:2:84939)
    at Au.t.$$.update (main.js:2:147307)
    at Q (main.js:2:4149)
    at new Eu (main.js:2:148158)
    at Pu (main.js:2:150286)
    at Cu (main.js:2:149612)
    at Object.p (main.js:2:151670)
    at F (main.js:2:2867)
    at D (main.js:2:2555)
```

```
Uncaught TypeError: Cannot read properties of undefined (reading 'start')
    at main.js:2:147330
    at ko (main.js:2:84939)
    at Au.t.$$.update (main.js:2:147307)
    at Q (main.js:2:4149)
    at new Eu (main.js:2:148158)
    at Pu (main.js:2:150286)
    at Cu (main.js:2:149612)
    at Object.p (main.js:2:151670)
    at F (main.js:2:2867)
    at D (main.js:2:2555)
```

**Comment by wilhelmklopp** [comment]
@cdunford thanks, appreciate it!

I just published version `2.1.3` of the VSCode extension – could you update to that version and then open the viz again to see if the issue still occurs?

**Comment by cdunford** [comment]
Issue is fixed in the new version. Thanks!
